web/css/modal.css

```diff
@@ -0,0 +1,14 @@
+.modal--content i {
+  margin-right: 20px;
+}
+
+.modal--content {
```

Comment:
the "modal-content" is a style also used for various menu (see sidebar) this should be generic style and not modal only